src/main/java/christmas/domain/menu/MenuName.java

```diff
@@ -0,0 +1,41 @@
+package christmas.domain.menu;
+
+import java.util.Objects;
+
+public class MenuName {
+
+    private final String name;
+
+    public MenuName(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        MenuName menuName = (MenuName) o;
+        return Objects.equals(name, menuName.name);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(name);
+    }
+
+    @Override
+    public String toString() {
+        return "MenuName{" +
+                "name='" + name + '\'' +
+                '}';
+    }
+
+}
```

Comment:
(추천해요!)

record를 사용하면 간결하게 표현할 수 있어요!

```suggestion
public record MenuName(String name){}
```